lib/Glimmer.pm

```diff
@@ -0,0 +1,494 @@
+# use strict;
+# use Data::Dumper;
+
+# my $out = Glimmer::call_genes_with_glimmer(file_with_path, { verbose => 1 });
+
+# print Dumper($out);
+
+package Glimmer;
+use strict;
+use File::Slurp;
+use File::Temp;
+use Data::Dumper;
+
+sub call_genes_with_glimmer
+{
+    my($contigs_file, $opts) = @_;
+
+    my $genetic_code     =   $opts->{genetic_code} || 11;   #...Default to the "standard" code
+
+    my $train            = qq();   #...Flag associated with '-train' switch
+    my $training_tbl     = qq();
+    my $training_contigs = qq();
```

Comment:
This syntax is pretty obscure, and if you just want to set these values to `""`, you are better off doing writing `""`. Unless someone is experienced and knowledgeable about perl (which many at KBase are not), they will have to go and look up what `qq()` does.